bika/lims/controlpanel/bika_sampletypes.py

```diff
@@ -5,216 +5,251 @@
+                "title": _("Prefix"),
+                "toggle": True}),
+            ("getMinimumVolume", {
+                "title": _("Minimum Volume"),
+                "toggle": True}),
+            ("RetentionPeriod", {
+                "title": _("Retention Period"),
+                "toggle": True}),
+            ("SampleMatrix", {
+                "title": _("SampleMatrix"),
+                "toggle": True}),
+            ("ContainerType", {
+                "title": _("Default Container"),
+                "toggle": True}),
+            ("getSamplePoints", {
+                "title": _("Sample Points"),
+                "toggle": True}),
+        ))
 
         self.review_states = [
-            {'id':'default',
-             'title': _('Active'),
-             'contentFilter': {'inactive_state': 'active'},
-             'transitions': [{'id':'deactivate'}, ],
-             'columns': ['Title',
-                         'Description',
-                         'getHazardous',
-                         'RetentionPeriod',
-                         'SampleMatrix',
-                         'ContainerType',
-                         'getSamplePoints',
-                         'getPrefix',
-                         'getMinimumVolume']},
-            {'id':'inactive',
-             'title': _('Dormant'),
-             'contentFilter': {'inactive_state': 'inactive'},
-             'transitions': [{'id':'activate'}, ],
-             'columns': ['Title',
-                         'Description',
-                         'getHazardous',
-                         'RetentionPeriod',
-                         'SampleMatrix',
-                         'ContainerType',
-                         'getSamplePoints',
-                         'getPrefix',
-                         'getMinimumVolume']},
-            {'id':'all',
-             'title': _('All'),
-             'contentFilter':{},
-             'columns': ['Title',
-                         'Description',
-                         'getHazardous',
-                         'RetentionPeriod',
-                         'SampleMatrix',
-                         'ContainerType',
-                         'getSamplePoints',
-                         'getPrefix',
-                         'getMinimumVolume']},
+            {
+                "id": "default",
+                "title": _("Active"),
+                "contentFilter": {"inactive_state": "active"},
+                "transitions": [{"id": "deactivate"}, ],
+                "columns": self.columns,
+            }, {
+                "id": "inactive",
+                "title": _("Dormant"),
+                "contentFilter": {"inactive_state": "inactive"},
+                "transitions": [{"id": "activate"}, ],
+                "columns": self.columns,
+            }, {
+                "id": "all",
+                "title": _("All"),
+                "contentFilter": {},
+                "columns": self.columns,
+            }
```

Comment:
This active/inactive states are always repeating everywhere. We could consider the opposite approach: make these active/inactive/all `review_states` generic everywhere and remove them from those few lists (if any) that do not apply